.vscode/extensions.json

```diff
@@ -0,0 +1,69 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+	// List of extensions which should be recommended for users of this workspace.
+	"recommendations": [
+		//General recommandations
+		"aaron-bond.better-comments", 	     		//Better Comments
+		"esbenp.prettier-vscode",		     		//Prettier
+		"streetsidesoftware.code-spell-checker",	//Code Spell Checker
+		"dbaeumer.vscode-eslint",		 		  	//ESLint
+		"wayou.vscode-todo-highlight",			    //Todo Highlight	
+
+		// To contribute with Golang
+		"golang.go",                         		// Go language support
+
+		//To contribute with python
+		"ms-python.python",	                 		//Python
+		"ms-python.black-formatter",         		//To format the code with black
+		"slightc.pip-manager",               		//To install pip-free packages
+		"ms-python.isort",							//To sort the imports
+		"the-compiler.python-tox",					//To run the tests
+		"ms-python.pylint",							//To run the linter
+
+		//To contribute with docker
+		"ms-azuretools.vscode-docker",       		//Docker support
+		"ms-vscode-remote.remote-containers",		//To run the containers remotely with docker-machine and docker-compose 
+
+		//To contribute with HTML
+		"formulahendry.auto-close-tag",      		//To automatically close the tag when you type a tag name.
+		"formulahendry.auto-rename-tag",     		//To automatically rename the tag when you type a tag name.
+		"cstrap.flask-snippets", 		     		//To add snippets for flask framework in HTML editor.
+		"ecmel.vscode-html-css",			 		//To add snippets for html and css 
+		"zignd.html-css-class-completion",			//To add snippets for html and css classes completion
+		"solnurkarim.html-to-css-autocompletion",	//To add snippets for html to css autocompletion 
```

Comment:
All HTML extensions can be removed, those are not mandatory
Those are nice to have for the dev if you want them

Reply:
ok, I will remove them